profile/profile.go

```diff
@@ -30,6 +30,14 @@ import (
 	"time"
 )
 
+// DiffBaseLabelKey is the key for the label which is used to mark profile
+// samples which are a part of the diff base profile.
+const DiffBaseLabelKey = "pprof::base"
```

Comment:
With DiffBaseSample function exposed, does the constants really need to be public?

Reply:
These don't need to be exposed. Done.